src/fsharp/FSharp.Core/string.fsi

```diff
@@ -85,9 +86,32 @@ namespace Microsoft.FSharp.Core
         /// <param name="str">The input string.</param>
         ///
         /// <returns>The resulting string.</returns>
+        /// 
+        /// <example> Filtering out just alphanumeric characters or just digits
+        /// <code lang="fsharp">
+        /// String.filter System.Uri.IsHexDigit "0 1 2 3 4 5 6 7 8 9 a A m M"  // evaluates "123456789aA"
+        /// String.filter System.Char.IsDigit "hello"                          // evaluates ""
+        /// </code>
```

Comment:
Similar to comments above 
* separate the examples by blank lines
* if any of the inputs are longer then ~5chars, then bind each of the inputs to `inp1` etc 
* open `System` if needed
* use pipeline